### ISSUE TITLE
Internal error: command not executed

### ISSUE THREAD

**Issue by lmarkov** [ghantoos/lshell#146]
Hello,
How can I execute command as:

```
find . -regextype posix-extended -regex '.*([a-zA-Z][0-9][0-9][.]php)$'
```

I removed this values in the configuration file but that did't help:

```
forbidden   : [';', '&', '|','`','>','<', '$(', '${']
```

**Comment by ghantoos** [comment]
Hi @lmarkov,

This seems to be due to the parser in lshell: it fails to properly interpret the regex content. I will try to look into this soon. It might be related to the lexer/parser issue that has been brought up to the frontpage lately.

**Comment by ghantoos** [comment]
This has been corrected in PR #236 

There is no need to clear the forbidden list. The forbidden characters should now be recognized as being quoted and not block the command.

```
$ grep -E '(allowed|forbidden)' etc/test.conf | grep -vE "^#"
allowed         : ['echo', 'ls', 'grep', 'cat', 'vi', 'find']
forbidden       : [';', '&', '|','`','>','<', '$(', '${']
```

Tested your command:
```
./bin/lshell --config etc/test.conf 
You are in a limited shell.
Type '?' or 'help' to get the list of allowed commands
ghantoos:~$ find . -regextype posix-extended -regex '.*([a-zA-Z][0-9][0-9][.]php)$'
./.vscode/extensions/bmewburn.vscode-intelephense-client-1.12.6/node_modules/intelephense/lib/stub/mongodb/BSON/Int64.php
ghantoos:~$
```
